test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Binders/SelectPathSegmentTokenBinderTests.cs

```diff
@@ -159,19 +159,14 @@ public void ValidateThrowInFindOperationsByBindingParameterTypeHierarchyExceptio
             SelectPathSegmentTokenBinder.TryBindAsOperation(new SystemToken("foo", null), model, entityType, out foundPathSegment).Should().BeFalse();
         }
 
-        [Fact(Skip = "This test currently fails.")]
+        [Fact]
         public void ValidateNonCatchableExceptionsThrowInFindOperationsByBindingParameterTypeHierarchyExceptionAndSurfaces()
         {
-            var model = new FindOperationsByBindingParameterTypeHierarchyThrowingStackOverflowEdmModel();
+            var model = new FindOperationsByBindingParameterTypeHierarchyThrowingNonCatchableExceptionEdmModel();
             IEdmEntityType entityType = new EdmEntityType("n", "EntityType");
             ODataPathSegment foundPathSegment = null;
             Action test = () => SelectPathSegmentTokenBinder.TryBindAsOperation(new SystemToken("foo", null), model, entityType, out foundPathSegment);
-
-#if NETCOREAPP1_0

```

Comment:
This #if is here because StackOverflowException is not in NetCore but the product code ignore StackOverflowException anyway and only responds to OutOfMemoryException, which is in both platforms, hence no need for #if.